frontend/elements/src/core/popups/popup.ts

```diff
@@ -0,0 +1,77 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+
+
+export type Color = "peach" | "green" ;
+export type Size ="medium" | "large";
+
+
+@customElement('template-popups')
```

Comment:
naming convention is typically to go from more general to more specific, and it should follow the folder structure. So in this case: `popups-template`, e.g. we may have other kinds of popups.

this isn't an absolute rule but I think it makes sense here

Reply:
Fixed